src/rmp/src/Restructure.cpp

```diff
@@ -322,7 +355,7 @@ void Restructure::getEndPoints(sta::PinSet& ends,
   auto sta_state = open_sta_->search();
   int path_count = 100000;
   float min_slack = area_mode ? -sta::INF : -sta::INF;
-  float max_slack = area_mode ? sta::INF : 0;
+  float max_slack = area_mode ? sta::INF : 5e-09;
```

Comment:
why 5e-9?

Reply:
Was to catch +ve slack designs for testing. Now made zero. Would need SDC tightening for this to trigger.